backend/auth/auth-model/src/main/java/com/meemaw/auth/sso/model/SsoSession.java

```diff
@@ -19,16 +19,16 @@
   // Number of characters in cookie
   public static final int SIZE = 50;
 
-  public static NewCookie cookie(String sessionId) {
-    return newCookie(sessionId, TTL);
+  public static NewCookie cookie(String sessionId, String domain) {
+    return newCookie(sessionId, domain, TTL);
   }
 
-  public static NewCookie clearCookie() {
-    return newCookie(null, 0);
+  public static NewCookie clearCookie(String domain) {
+    return newCookie(null, domain, 0);
   }
 
-  private static NewCookie newCookie(String sessionId, int maxAge) {
-    return new NewCookie(COOKIE_NAME, sessionId, COOKIE_PATH, null, null, maxAge, false);
+  private static NewCookie newCookie(String sessionId, String domain, int maxAge) {
+    return new NewCookie(COOKIE_NAME, sessionId, COOKIE_PATH, domain, null, maxAge, false);
```

Comment:
cookies should probably be marked secure in prod.

Reply:
should be marked secure when user comes via https (which should be in all environments except localhost)